pac4j-oauth/src/main/java/org/pac4j/oauth/redirect/OAuth20RedirectActionBuilder.java

```diff
@@ -53,7 +53,7 @@ public RedirectAction redirect(final WebContext context) throws HttpAction {
 
                 service = this.configuration.getService();
             }
-            final String authorizationUrl = service.getAuthorizationUrl(this.configuration.getCustomParams());
+            final String authorizationUrl = service.getAuthorizationUrl();
```

Comment:
The parameter in getAuthorizationUrl are used to add additional parameters when building the authorization url by the scribe api implementation, as in org.pac4j.scribe.builder.api.GenericApi20, so customParameters should not be used here.